cEP-0009.md

```diff
@@ -0,0 +1,209 @@
+| Metadata     |                                                          |
```

Comment:
The commit message could use a bit of refining (just mention that this is about quickstart in the commit body).